components/Bottom.js

```diff
@@ -0,0 +1,138 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+export default function Footer() {
+  const { pathname } = useRouter();
+
+  return (
+    <StyledNavigation>
+      <NavLink href={`/spots`}>
+        <StyledNavItem>
+          {pathname === "/spots" ? (
+            <svg
```

Comment:
I would consider making a `Icons` directory inside your `components` directory.  Something like this. 

``` 
 📁 components
  📁 Icons
    Spot.js
    SpotActive.js
    Create.js
    CreateActive.js
    Favourite.js
    FavouriteActive.js
```

Then you could export each icon and import them into this file. e.g. for Spots

``` js
export default function SpotIcon() {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      height="24px"
      viewBox="0 0 24 24"
      width="24px"
      fill="#ffffff"
    >
      <path d="M0 0h24v24H0V0z" fill="none" />
      <path d="M4 10.5c-.83 0-1.5.67-1.5 1.5s.67 1.5 1.5 1.5 1.5-.67 1.5-1.5-.67-1.5-1.5-1.5zm0-6c-.83 0-1.5.67-1.5 1.5S3.17 7.5 4 7.5 5.5 6.83 5.5 6 4.83 4.5 4 4.5zm0 12c-.83 0-1.5.68-1.5 1.5s.68 1.5 1.5 1.5 1.5-.68 1.5-1.5-.67-1.5-1.5-1.5zM7 19h14v-2H7v2zm0-6h14v-2H7v2zm0-8v2h14V5H7z" />
    </svg>
  )
}
```

Which means Footer would look something like this

``` js
...
import SpotIcon from "./Icons/Spot";
import SpotIconActive from "./Icons/SpotActive";
...

<NavLink href={`/spots`}>
  <StyledNavItem>
    {pathname === "/spots" ? <SpotIconActive /> : <SpotIcon />}
    <NavIconText>Spots</NavIconText>
  </StyledNavItem>
</NavLink>
```

None of the functionality will change... it will just be potentially easier to read and maintain in the future.

Reply:
Thank you for your tip! I will figure it out!